wfe2/wfe_test.go

```diff
@@ -3276,3 +3276,19 @@ func TestGETAPIChallenge(t *testing.T) {
 		}
 	}
 }
+
+func TestGetAPIAndMandatoryPOSTAsGET(t *testing.T) {
+	wfe, _ := setupWFE(t)
+	makeGet := func(path, endpoint string) (*http.Request, *web.RequestEvent) {
+		return &http.Request{URL: &url.URL{Path: path}, Method: "GET"},
+			&web.RequestEvent{Endpoint: endpoint, Extra: map[string]interface{}{}}
+	}
+	features.Set(map[string]bool{"MandatoryPOSTAsGET": true})
```

Comment:
`errcheck` is failing your build here. I think you need to `_ =` it or check the err.